src/RendererDX11.cpp

```diff
@@ -141,9 +145,96 @@ ID3D11ShaderResourceView* RendererDX11::get_render_texture_view() const
     return m_render_target_texture_view;
 }
 
+void RendererDX11::render_shadow_map() const
+{
+    get_instance_dx11()->bind_dsv_for_shadow_mapping();
+    shadow_shader->use();
+
+    for (auto const& shader : m_shaders)
+    {
+        for (auto const& material : shader->materials)
+        {
+            if (material->is_gpu_instanced)
+            {
+                draw_instanced(material, glm::mat4(1.0f), glm::mat4(1.0f));
```

Comment:
GPU instancing doesn't work but if it did, this wouldn't work either. Let's just leave a comment that this doesn't work and leave `std::unreachable()`